src/Sulu/Bundle/AudienceTargetingBundle/Tests/Unit/Controller/TargetGroupEvaluationControllerTest.php

```diff
@@ -127,16 +125,14 @@ public function testTargetGroupHitAction($oldTargetGroup, $newTargetGroup, $newT
         $targetGroupEvaluationController->targetGroupHitAction();
     }
 
-    public function provideTargetGroupHit()
+    public static function provideTargetGroupHit(): array
     {
-        $oldTargetGroup1 = $this->prophesize(TargetGroupInterface::class);
-        $oldTargetGroup1->getId()->willReturn(1);
-        $newTargetGroup1 = $this->prophesize(TargetGroupInterface::class);
-        $newTargetGroup1->getId()->willReturn(2);
+        $oldTargetGroup1 = self::createConfiguredStub(TargetGroupInterface::class, ['getId' => 1]);
```

Comment:
we will keep prophesize for mocking

Reply:
The problem is that in PhpUnit 10 the dataprovider have to be static and `prophesize` is not a static method.